### PR TITLE
feat: add multi-select dropdown for verification status of captures f…

### LINKED ISSUE #114: Create multi-select dropdown for Verification Status so that Captures view total counts are correct
Total number of filtered not showing (only displaying total number of all trees) see screenshot
Screenshot 2021-06-11 at 10 48 43

It should be around 2000 captures in this case. 
We could also rename that entire section to captures.

### PR DESCRIPTION
**Pull request by tranquanghuy0801**
## Description

Add Multi-select dropdown for Verification Status field in filter of Captures page

**Issue(s) addressed**
- Resolves #114 

**What kind of change(s) does this PR introduce?** 
- [x] Enhancement
- [ ] Bug fix
- [ ] Refactor

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Issue

**What is the current behavior?**

The current design is a single dropdown menu for Verification Status field in Captures page

**What is the new behavior?**

The new design is a multi-select dropdown menu for Verification Status field in Captures page

## Breaking change
**Does this PR introduce a breaking change?** 

No breaking changes

## Video Screenshot 

https://user-images.githubusercontent.com/29462498/143463655-7004774c-67bd-4376-a601-d3e6d4994ca7.mov

### CODE REVIEW

File: src/components/CaptureFilter.js [hunk 1]
```diff
@@ -119,6 +132,59 @@ function Filter(props) {
...
+      setActive(undefined);
+      setApproved(undefined);
+      return;
+    }
+    handleChangeActiveApproved(value);
+    setVerificationStatus(value);
+  };
+
+  // set active and approved field based on the checkbox values
+  function handleChangeActiveApproved(value) {
```
Comment (nmcharlton) [inline src/components/CaptureFilter.js:155]: Unfortunately, this logic doesn't cover all possible combinations, and it's hard to follow what it's trying to do.
If _Approved_ and _Rejected_ are selected, everything (including _Awaiting Verification_) is included.

I'd suggest replacing `approved` and `active` in the filter model with these Verification States and only converting when the API query is constructed.
You will need to apply an `and` clause to the query object: https://loopback.io/doc/en/lb4/Where-filter.html#and--or

**Review by nmcharlton — CHANGES_REQUESTED** [review]

**Comment by nmcharlton** [comment]
Hey @tranquanghuy0801. What's the latest on this PR? I notice you've pushed a couple of commits, but the problem with the logic is still present. Do you need some help working out what needs to change?

**Comment by tranquanghuy0801** [comment]
@nmcharlton The latest PR is just resolving some issues when pulling the updates from master branch. I understand how to fix and am working on this. Sorry for the delay.

**Comment by tranquanghuy0801** [comment]
@nmcharlton I have updated the code as you suggested. Now you can check my work. Thanks

**Review by nmcharlton — CHANGES_REQUESTED** [review]
@tranquanghuy0801 This seems to be working well, but I don't think it makes sense to be able to select none of the values (and get zero results).

Having nothing selected should return the same results as having all values selected (i.e., all captures).

**Comment by tranquanghuy0801** [comment]
> @tranquanghuy0801 This seems to be working well, but I don't think it makes sense to be able to select none of the values (and get zero results).
> 
> Having nothing selected should return the same results as having all values selected (i.e., all captures).

Yeah I intended to show all results when the verification status is empty. It turned out it has an error in the function onHandleChange the value of these statues, which causes the incorrect data on the table. Now I fix that one.

**Review by nmcharlton — COMMENTED** [review]
Thanks @tranquanghuy0801!

**Review by nmcharlton — APPROVED** [review]
